Parkduksung/ElevenStreetSearchExample/app/src/main/java/com/exam/elevenstreet/view/product/ProductActivity.kt

```diff
@@ -26,38 +26,38 @@ class ProductActivity : AppCompatActivity() {
             search_button.setOnClickListener {
 
                 var inputkeyword = "${search_text.text}"
-                ProductRemoteDataSource()
-                    .getProductlist(inputkeyword) { productList ->
-                        runOnUiThread {
-                            recyclerview_product.run {
-                                var adapter =
-                                    ProductAdapter(productList as ArrayList<ProductResponse>)
-                                this.adapter = adapter
-                                layoutManager = LinearLayoutManager(this@ProductActivity)
+
+                ProductRemoteDataSource().getProductlist(inputkeyword,
```

Comment:
```suggestion
                ProductRemoteDataSource().getProductlist(
                                    "${search_text.text}",
```
inputkeyword 변수를 재사용하지 않는다면 굳이 변수를 만들지 않아도 될것같아요.

Reply:
bff4f10 변수를 없애버리고 바로 그냥 넣어버려씀